roles/candlepin/tasks/main.yml

```diff
@@ -77,6 +77,8 @@
       - |
         [Install]
         WantedBy=default.target
+    healthcheck: curl --fail --insecure https://localhost:23443/candlepin/status
+    sdnotify: healthy
```

Comment:
https://docs.ansible.com/ansible/latest/collections/containers/podman/podman_container_module.html#parameter-sdnotify doesn't document this option, but `sdnotify` maps directly to `Notify=` and https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html#notify-defaults-to-false does document that it uses the `HealthCmd` option (which is what the above `healthcheck` is for).

Reply:
https://github.com/containers/ansible-podman-collections/pull/911